src/commands/init/index.ts

```diff
@@ -79,4 +81,48 @@ export default class Init extends Command {
 
     ux.action.stop();
   }
+
+  private async getStartAppMessage(): Promise<string> {
+    const docsUrl = 'https://docs.demo.saas.apptoku.com/';
```

Comment:
Please move this to the `config/index.ts`

Reply:
changes applied